tutorials/haskell/dotenv-hs/tutorial.md

```diff
@@ -0,0 +1,284 @@
+---
+title: Setting up your environment with dotenv
+published: not-yet
+ghc: 8.0.2
+lts: 8.4
+libraries: dotenv-0.4.0.0
+language: haskell
+author-name: Cristhian Motoche
+description: Check Out the two working modes of dotenv to set up an
+environment before the execution of an application.
+---
+# Introduction
+## What is dotenv?
+[Dotenv][dotenv] is a Haskell library and an Standalone tool that enables to set
+up the configuration of an environment before the execution of an application.
+As it is suggested in [The Twelve Factor App](https://12factor.net/config)
+the configuration should be separated from the code. A good way to do that is
+by loading **environment variables** (env vars) and `dotenv` is the perfect tool
+for that, because it loads all the needed env vars for an application to run.
+
+## Why it is important?
+The configuration should be separated from the code for one simple reason: the
+code should be the same in every environment (development, testing, staging,
+quality assurance, production, etc.) and configuration is the opposite, it
+changes in every environment.
+
+Loading env vars is a good way to set up the configuration of an application,
+because they can change easily everytime the environment changes and without
+changing code.
+
+## How it works?
+Dotenv reads the needed application's env vars defined in a *dotenv example file*
```

Comment:
I think we can leave it like *dotenv example file*, because it is a general definition. As you know, in `dotenv` the file doesn't need to be called *.env.example* strictly. Let me know what you think.